.github/workflows/emscripten.yml

```
@@ -0,0 +1,70 @@
+# Attributed to NumPy via https://github.com/numpy/numpy/pull/25894
+# https://github.com/numpy/numpy/blob/d2d2c25fa81b47810f5cbd85ea6485eb3a3ffec3/.github/workflows/emscripten.yml
+name: Test Emscripten/Pyodide build
+
+on:
+  # TODO: remove after ready to merge
+  push:
+    # branches: [ 'main']
+  pull_request:
+    # branches: [ 'main' ]
+  workflow_dispatch:
+
+# Cancel intermediate runs if a new run is queued
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  build-wasm-emscripten:
+    name: Build statsmodels distribution for Pyodide
+    runs-on: ubuntu-latest
+    env:
+      PYODIDE_VERSION: 0.26.0
```

```suggestion
      PYODIDE_VERSION: 0.26.1
```

Pyodide version 0.26.1 is out now